server/index.js

```diff
@@ -37,8 +37,14 @@ app.get('/connection', protectRoute, (req, res) => {
         return;
       }
       if (socket) {
-        socket.emit('token', createToken(foundUser.pet_name || 'Pet'));
-        res.json({ token: createToken(foundUser.name) });
+        socket.emit({
```

Comment:
you removed 'token' which is the connection to the pet client. if you check media.jsx, socket is listening .on('token', ...)